types/googlepay/googlepay-tests.ts

```diff
@@ -101,6 +101,14 @@ function addGooglePayButton() {
     // $ExpectError
     buttonOptions.buttonRootNode = document.createElement('div');
 
+    buttonOptions.buttonLocale = '';
```

Comment:
Would it be useful to add a correct locale in the tests?

Reply:
`'en'` and `'zh'` are examples of _real_ values. `''` and `'qw'` are valid according to the type definition, but unless they're one of the supported languages, it will fallback to the browser/operating system locale.